api/manifest/manifest.go

```diff
@@ -278,9 +305,106 @@ func (me *ManifestEntry) EnvironmentArray() []string {
 	return arr
 }
 
-func (m *Manifest) MissingEnvironment() []string {
+// NOTE: this is the simpler approach:
+//       build up the ENV from the declared links
+//       assuming local dev is done on DOCKER_HOST
+func (me *ManifestEntry) ResolvedLinkVars() (map[string]string, error) {
+	linkVars := make(map[string]string)
+
+	if me.Manifest == nil {
+		return linkVars, nil
+	}
+
+	for _, link := range me.Links {
+		other := (*me.Manifest)[link]
```

Comment:
@ddollar this is what i need that backreference for.  the methods calling ResolvedLinkVars are also defined on `ManifestEntry`, so I can't get it there either. I *could* pass the manifest to `ResolvedEnvironment` in the call to `MissingEnvironment`, but would also have to modify `runAsync` to take the manifest and pass that through... I'll wire this all up in another PR so you can tell me what you think.